app/package.json

```diff
@@ -13,6 +13,7 @@
     "next": "10.0.7",
     "next-connect": "^0.10.0",
     "react": "17.0.1",
-    "react-dom": "17.0.1"
+    "react-dom": "17.0.1",
+    "mongoose": "latest"
```

Comment:
Can we set this to a specific version number? It can be the latest one, but we don't want unnecessary conflicts when redeploying later